packages/account/src/Configs/user-profile-validation-config.ts

```diff
@@ -0,0 +1,186 @@
+            .max(70, localize('Only 70 characters, please.'))
+            .matches(
+                address,
+                `${localize('Use only the following special characters:')} ${addressPermittedSpecialCharacters}`
+            )
+            .when({
+                is: () => is_svg,
+                then: Yup.string().test(
+                    'po_box',
+                    localize('P.O. Box is not accepted in address'),
+                    value => !postalOfficeBoxNumber.test(value ?? '')
+                ),
+            }),
+        address_line_2: Yup.string()
+            .max(70, localize('Only 70 characters, please.'))
+            .matches(
+                address,
+                `${localize('Use only the following special characters:')} ${addressPermittedSpecialCharacters}`
+            )
+            .when({
+                is: () => is_svg,
+                then: Yup.string().test(
+                    'po_box',
+                    localize('P.O. Box is not accepted in address'),
+                    value => !postalOfficeBoxNumber.test(value ?? '')
+                ),
+            }),
+        address_postcode: Yup.string()
+            .max(20, localize('Please enter a postal/ZIP code under 20 characters.'))
+            .matches(postalCode, localize('Only letters, numbers, space and hyphen are allowed.')),
+        address_state: Yup.string().matches(addressState, localize('State is not in a proper format')),
+    });
+
+export const getPersonalDetailsBaseValidationSchema = (broker_code?: string) =>
+    Yup.object({
+        salutation: Yup.string().when({
+            is: () => broker_code === 'maltainvest',
+            then: Yup.string().required(localize('Salutation is required.')),
+        }),
+        account_opening_reason: Yup.string().required(localize('Account opening reason is required.')),
+        first_name: Yup.string()
+            .required(localize('First name is required.'))
+            .max(50, localize('Enter no more than 50 characters.'))
+            .matches(name, localize('Letters, spaces, periods, hyphens, apostrophes only.')),
+        last_name: Yup.string()
+            .required(localize('Last name is required.'))
+            .max(50, localize('Enter no more than 50 characters.'))
+            .matches(name, localize('Letters, spaces, periods, hyphens, apostrophes only.')),
+        date_of_birth: Yup.string()
+            .required('Date of birth is required.')
+            .test({
+                name: 'validate_dob',
+                test: value => dayjs(value).isValid() && dayjs(value).isBefore(dayjs().subtract(18, 'years')),
+                message: localize('You must be 18 years old and above.'),
+            }),
+        phone: Yup.string()
+            .required(localize('Phone is required.'))
+            .min(9, localize('You should enter 9-35 numbers.'))
+            .max(35, localize('You should enter 9-35 characters.'))
```

Comment:
this is updated to 20 numbers. @likhith-deriv in phone number verification project
@suisin-deriv  lets keep an eye on this to avoid getting validations override when this PR is merged

Reply:
In that case can we have that come from util library?

Reply:
So the message states, You should enter 9-20 numbers ? @amina-deriv , @suisin-deriv